cypress-io/cypress
# dependency: update loader-utils to 1.4.2

- Closes https://github.com/cypress-io/cypress/issues/28208

### Additional details

Resolves critical vulnerability [CVE-2022-37601](https://nvd.nist.gov/vuln/detail/CVE-2022-37601) by temporarily removing the section for `loader-utils@1.4.0` from [yarn.lock](https://github.com/cypress-io/cypress/blob/develop/yarn.lock) and allowing `yarn` to reevaluate the dependencies.

This causes Yarn to add [loader-utils@1.4.2](https://github.com/webpack/loader-utils/releases/tag/v1.4.2) to [yarn.lock](https://github.com/cypress-io/cypress/blob/develop/yarn.lock) instead of the vulnerable [loader-utils@1.4.0](https://github.com/webpack/loader-utils/releases/tag/v1.4.0).

### Steps to test

Execute the following and ensure that [loader-utils@1.4.0](https://github.com/webpack/loader-utils/releases/tag/v1.4.0) is no longer configured:

```shell
git clean -xfd
yarn
yarn list --pattern loader-utils
```

```text
$ yarn list --pattern loader-utils
yarn list v1.22.22
warning Resolution field "pretty-format@26.4.0" is incompatible with requested version "pretty-format@29.4.3"
warning Resolution field "pretty-format@26.4.0" is incompatible with requested version "pretty-format@^27.0.2"
warning Resolution field "vue-template-compiler@2.6.12" is incompatible with requested version "vue-template-compiler@^2.7.14"
├─ adjust-sourcemap-loader@4.0.0
│  └─ loader-utils@2.0.3
├─ loader-utils@1.4.2
└─ resolve-url-loader@5.0.0
   └─ loader-utils@2.0.3
```

### How has the user experience changed?

The following security scan should no longer report [CVE-2022-37601](https://github.com/advisories/GHSA-76p3-8jx3-jpfq)

```shell
trivy image --ignore-unfixed --pkg-types library --severity CRITICAL cypress/included
```

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
